scripts/rollup/packaging.js

```diff
@@ -45,7 +47,15 @@ function getBundleOutputPaths(bundleType, filename, packageName) {
     case RN_PROD:
       switch (packageName) {
         case 'react-native-renderer':
-          return [`build/react-native/${filename}`];
+          return [`build/react-native/oss/${filename}`];
```

Comment:
Things in the "oss" sub directory are blacklisted from FB configs in order to prevent Haste module conflicts.